camelot/view/mainwindow.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, gui_context, parent=None, window=None):
         
         logger.debug('setting up workspace')
         self.gui_context = gui_context
-        self.workspace = DesktopWorkspace( self.app_admin, self.window )
```

Comment:
ik zou proberen self.window te verwijderen om hier geen referentie te hebben naar het QObject.

moet MainWindow nog afleiden van QObject ?

mss kunnen we MainWindow best hernoemen naar 'MainWindowBuilder' of zo, zodat het duidelijk is dat het geen window is :)

Reply:
MainWindow is nog steeds afgeleid van QObject omdat deze de Qt slots bevat. Deze slots gebruiken dan self.window om de veranderingen toe te passen op de QMainWindow (als een soort proxy object).

De huidige naamgeving is zeker verwarrend, maar mischien is MainWindowProxy duidelijker vermits het ook geen zuivere builder is?